scripts/runtests.py

```diff
@@ -99,19 +112,17 @@ def main() -> None:
             print("\nRunning stubtest...")
             stubtest_result = subprocess.run([sys.executable, "tests/stubtest_stdlib.py", stub])
         else:
-            run_stubtest_query = (
-                f"\nRun stubtest for {stub!r} (Y/N)?\n\n"
-                "NOTE: Running third-party stubtest involves downloading and executing arbitrary code from PyPI.\n"
-                f"Only run stubtest if you trust the {stub!r} package.\n"
-            )
-            run_stubtest_answer = input(colored(run_stubtest_query, "yellow")).lower()
-            while run_stubtest_answer not in {"yes", "no", "y", "n"}:
-                run_stubtest_answer = input(colored("Invalid response; please try again.\n", "red")).lower()
-            if run_stubtest_answer in {"yes", "y"}:
-                print("\nRunning stubtest.")
+            if run_stubtest:
                 stubtest_result = subprocess.run([sys.executable, "tests/stubtest_third_party.py", stub])
```

Comment:
```suggestion
                print("\nRunning stubtest...")
                stubtest_result = subprocess.run([sys.executable, "tests/stubtest_third_party.py", stub])
```

Motivation: at the moment, this is what's printed to the terminal if I run it with the `--run-stubtest` flag:

```
(.venv) C:\Users\alexw\coding\typeshed>python scripts/runtests.py stubs/requests --run-stubtest

Running pycln...
Looks good! ✨ 🍰 ✨
18 files left unchanged.

Running isort...

Running Black...
All done! ✨ 🍰 ✨
18 files left unchanged.

Running Flake8...

Running check_consistent.py...

Running check_new_syntax.py...

Running Pyright on Python 3.7...

Skipping Pyright tests: npx is not installed or can't be run!

Running mypy for Python 3.7...
*** Testing Python 3.7 on win32
Testing third-party packages...
testing requests (17 files)... success

--- success, 17 files checked ---
requests... Note: requests is not currently tested on win32 in typeshed's CI.
success

Skipping pytype on Windows. You can run the test with WSL.

Running regression tests for Python 3.7...
Running mypy --platform win32 --python-version 3.7 on the test cases for 'requests'... success

Test completed successfully!



--- TEST SUMMARY: All tests passed! ---

Flake8: Success
Check consistent: Success
Check new syntax: Success
Pyright: Skipped
mypy: Success
stubtest: Success
pytype: Skipped
Regression test: Success
```

Reply:
Agreed. This was there before (line 111)